CHANGELOG.md

```diff
@@ -1,3 +1,36 @@
+### pg_auto_failover v1.6.1 (July 7, 2021) ###
+
+This release contains monitor schema changes, so the monitor extension gets
+a version bump from 1.5 to 1.6, and this is the first release in the 1.6
+series.
+
+In this release we introduce a new state in the FSM: "dropped". This state
+allows a node to realise it's been dropped from the monitor, and act
+accordingly (mostly, stops cleanly and register it's been dropped).
```

Comment:
Should contain something along the lines of:

This means that the new `pg_autoctl drop node` will only complete when the node that is being dropped is still reachable. To drop a node that is unreachable (e.g. machine died), you should now use `pg_autoctl drop node --force`.

Reply:
Good point, added now.